content/source/docs/enterprise/private/rhel-install-guide.html.md

```diff
@@ -34,7 +33,7 @@ If you opt to use Docker from RHEL extras, then you must make a change to its de
 Sure! Just be sure to have at least 1.13.1 and authorization plugins disabled.
 
 ### When I run the installer, it allows me to download and install Docker CE on RedHat. Can I use that?
-Yes, Docker CE is compatible with the current installer; however it is not directly supported by RedHat. You still need to be sure that the storage backend is configured properly as by default, Docker will be using devicemapper in loopback, an entirely unsupported mode.
+Yes, Docker CE is compatible with the current installer; however it is not directly supported by RedHat (See https://access.redhat.com/articles/2726611). You still need to be sure that the storage backend is configured properly as by default, Docker will be using devicemapper in loopback, an entirely unsupported mode.
```

Comment:
"See" → "see".

Reply:
I'd suggest instead making this a true link, and changing the punctuation a bit:

```
Yes, Docker CE is compatible with the current installer. However, it is not directly [supported by RedHat](https://access.redhat.com/articles/2726611).
```

I'm also a little unsure what we're suggesting by saying that it's not directly supported...? What kind of hesitation should that give the customer, if we're still saying it's fine, ultimately?